std/ascii.d

```diff
@@ -102,6 +103,75 @@ enum LetterCase : bool
     assert(sha1HMAC == "49f2073c7bf58577e8c9ae59fe8cfd37c9ab94e5");
 }
 
+/++
+    All the control characters in ASCII table
++/
```

Comment:
Are the names of these enumerations standardized? Where do they come from?

You may want to add a reference to this documentation comment.

Reply:
Ref added. as for the abbreviation names, I think they are standartized because they are the same no matter where they are listed, but their descriptions are not. For example, horizonal tab vs. horizonal tabulation, line feed vs. newline.

On the other hand, if we decide to use full names instead, I could just define two names with same values in cases like that latter example.

Reply:
> As for the abbreviation names, I think they are standartized because they are the same no matter where they are listed, but their descriptions are not. For example, horizonal tab vs. horizonal tabulation, line feed vs. newline.

A little old (only about 5 decades), but both the abbreviations and the full names are standardised here: https://tools.ietf.org/html/rfc20#section-4.1.